pom.xml

```diff
@@ -8,8 +8,8 @@
     <packaging>jar</packaging>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
I think this can be kept to 1.8 from a Java language perspective.

Reply:
I agree, updated accordingly.